website_coupon/__init__.py

```diff
@@ -21,5 +21,5 @@
 #
 ##############################################################################
 
-import models
-import controllers
+import models  # noqa
+import controllers  # noqa
```

Comment:
this is not necessary. this flake8 error will be ignored by Travis.
Localy, you should create an exception on F401 for __init__.py files.

Reply:
Good point, thanks for the tip.